artemis-server/src/main/java/org/apache/activemq/artemis/core/postoffice/impl/WildcardAddressManager.java

```diff
@@ -175,36 +175,38 @@ public void clear() {
       wildCardAddresses.clear();
    }
 
-   private Address addAndUpdateAddressMap(final SimpleString address, boolean getBiased) {
+   private Address addAndUpdateAddressMap(final SimpleString address) {
       final boolean containsWildCard = address.containsEitherOf(wildcardConfiguration.getAnyWords(), wildcardConfiguration.getSingleWord());
       final Map<SimpleString, Address> addressMap = containsWildCard ? wildCardAddresses : addresses;
-      Address actualAddress = null;
-      if (getBiased) {
-         // CHM::get doesn't need to synchronize anything, so it's to be preferred for getBiased cases
-         actualAddress = addressMap.get(address);
-      }
+      Address actualAddress = addressMap.get(address);
       if (actualAddress == null) {
```

Comment:
it's just a personal thing with code: 
```java
if (actualAddress != null) {
    return actualAddress;
}
// from now one, slow path
```
It's helpful for the brain to not further reading the rest of code, if not needed 
and it allows to pack the slow path code into a specific (`synchronized`) method too

Reply:
true, that is better.

Reply:
I looked at changing this, if I follow the same pattern on the get check inside the sync I end up with a method with 3 return statements. At the moment it has a single return, which i like.
I thin the single return statement is better than 2 fast paths and one slow :-)